src/sagemaker/lineage/query.py

```diff
@@ -192,13 +440,117 @@ def __init__(
         """
         self.edges = []
         self.vertices = []
+        self.startarn = []
 
         if edges is not None:
             self.edges = edges
 
         if vertices is not None:
             self.vertices = vertices
 
+        if startarn is not None:
+            self.startarn = startarn
+
+    def __str__(self):
+        """Define string representation of ``LineageQueryResult``.
+
+        Format:
+        {
+            'edges':[
+                {'source_arn': 'string', 'destination_arn': 'string', 'association_type': 'string'},
+                ...],
+
+            'vertices':[
+                {'arn': 'string', 'lineage_entity': 'string', 'lineage_source': 'string',
+                '_session': <sagemaker.session.Session object>},
+                ...],
+
+            'startarn':['string', ...]
+        }
+
+        """
+        return (
+            "{"
+            + "\n\n".join("'{}': {},".format(key, val) for key, val in self.__dict__.items())
+            + "\n}"
+        )
+
+    def _covert_edges_to_tuples(self):
+        """Convert edges to tuple format for visualizer."""
+        edges = []
+        # get edge info in the form of (source, target, label)
+        for edge in self.edges:
+            edges.append((edge.source_arn, edge.destination_arn, edge.association_type))
+        return edges
+
+    def _covert_vertices_to_tuples(self):
+        """Convert vertices to tuple format for visualizer."""
+        verts = []
+        # get vertex info in the form of (id, label, class)
+        for vert in self.vertices:
+            if vert.arn in self.startarn:
+                # add "startarn" class to node if arn is a startarn
+                verts.append((vert.arn, vert.lineage_source, vert.lineage_entity, True))
+            else:
+                verts.append((vert.arn, vert.lineage_source, vert.lineage_entity, False))
+        return verts
+
+    def _get_visualization_elements(self):
+        """Get elements(nodes+edges) for visualization."""
+        verts = self._covert_vertices_to_tuples()
+        edges = self._covert_edges_to_tuples()
+
+        elements = {"nodes": verts, "edges": edges}
+        return elements
+
+    def visualize(self, path="pyvisExample.html"):
```

Comment:
* nit add type decorations i.e. `path: Optional[str] = "pyvisExample.html"`
* is that the best default arg? could be something more domain specific like `lineage_graph.html`